test/states/home_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:farmsmart_flutter/redux/home/home_state.dart';
+import 'package:farmsmart_flutter/redux/home/home_reducer.dart';
+import 'package:farmsmart_flutter/redux/home/home_actions.dart';
+import 'package:farmsmart_flutter/model/loading_status.dart';
+import 'package:farmsmart_flutter/redux/home/screens.dart';
+
+
+void main() {
+  test('Navigation test - to my plot', (){
```

Comment:
I see you can use group() to wrap a set of tests. Good to set the convention for tests early, as such could we include some documentation in the README - can reference external style.

e.g.
```
// unit
group("My function", () {
  test("should do something") {}
  test("should not do something else {}
});
```

Reply:
Done